docs/next/guides/building-and-testing/building.md

```diff
@@ -9,83 +9,263 @@ tags:
+     - The JavaScript package
+     - The Angular package
+     - The React package
+     - The Vue package
+     - A code examples package
+
+#### Root directory build tasks
+
+You can also run individual root directory `build` tasks:
+
+::: details Root directory build tasks
+`build:codesandbox`
+  - PLACEHOLDER
+:::
+
+### Building the JavaScript package
+
+To build the JavaScript package:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to `./handsontable`.
+3. Run `npm run build`.<br>Only the JavaScript package builds.
+
+To build the JavaScript package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in handsontable build`.<br>Only the JavaScript package builds.
+
+#### JavaScript build tasks
+
+You can also run individual JavaScript `build` tasks:
+
+::: details JavaScript build tasks
+`npm run build:commonjs`
+  - Transpiles files into the `CommonJS` format.
+  - During release, these files are published into the npm repository.
+
+`npm run build:es`
+  - Transpiles files into the `import/export` format.
+  - During release, these files are published into the npm repository.
+
+`npm run build:umd`
+  - Creates the following bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.js`
+    - `dist/handsontable.css`
+    - `dist/handsontable.full.js`
+    - `dist/handsontable.full.css`
+
+`npm run build:umd.min`
+  - Creates the minified bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.min.js`
+    - `dist/handsontable.min.css`
+    - `dist/handsontable.min.full.js`
+    - `dist/handsontable.min.full.css`
+
+`npm run build:walkontable`
+  - PLACEHOLDER
+
+`npm run build:languages`
+  - Creates the [language](@/guides/internationalization/internationalization-i18n.md) bundles compatible with the Universal Module Definition, for example:
+    - `dist/languages/de-DE.js`
+    - `dist/languages/all.js`
+
+`build:languages.es`
+  - PLACEHOLDER
+
+`npm run build:languages.min`
+   - Creates the minified [language](@/guides/internationalization/internationalization-i18n.md) bundles compatible with the Universal Module Definition, for example:
+     - `dist/languages/de-DE.min.js`
+     - `dist/languages/all.min.js`
+:::
+
+### Building the Angular package
+
+To build the Angular package:
+1. Make sure you meet the [building requirements](#building-requirements).
+3. Go to `./wrappers/angular`.
+4. Run `npm run build`.<br>Only the Angular package builds.
+
+To build the Angular package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in angular build`.<br>Only the Angular package builds.
+
+#### Angular build tasks
+
+You can also run individual Angular `build` tasks:
+
+::: details Angular build tasks
+`prebuild`
+  - PLACEHOLDER
```

Comment:
I'd skip `prebuild` and `postbuild` script description, as they're not meant to be ever called separately - they'll be automatically run before and after the `build` script.

Reply:
Done: 3528669